contrib/graphite/blueflood.py

```diff
@@ -51,6 +51,7 @@ def calc_res(start, stop):
 
```

Comment:
it looks like this is the same as the commit in this PR: https://github.com/rackerlabs/blueflood/pull/565